Memory reports do not work when Nordic proprietary LL is selected

**Describe the bug**
```
pdunaj@ubuntu:~/work/ncs/nrf/applications/nrf_desktop$ time ( B=nrf52840_pca20041; cd build-$B && ninja rom_report )
[1/2] cd /home/pdunaj/work/ncs/nrf/applications/nrf_desktop/build-nrf52840_pca20041/zephyr/cmake...hyr-eabi-nm -o /home/pdunaj/work/ncs/nrf/applications/nrf_desktop/build-nrf52840_pca20041/zephy
FAILED: zephyr/cmake/reports/CMakeFiles/rom_report 
cd /home/pdunaj/work/ncs/nrf/applications/nrf_desktop/build-nrf52840_pca20041/zephyr/cmake/reports && /usr/bin/python3 /home/pdunaj/work/ncs/zephyr/scripts/footprint/size_report -F --objdump /opt/zephyr-sdk/arm-zephyr-eabi/bin/arm-zephyr-eabi-objdump --objcopy /opt/zephyr-sdk/arm-zephyr-eabi/bin/arm-zephyr-eabi-objcopy --nm /opt/zephyr-sdk/arm-zephyr-eabi/bin/arm-zephyr-eabi-nm -o /home/pdunaj/work/ncs/nrf/applications/nrf_desktop/build-nrf52840_pca20041/zephyr
Traceback (most recent call last):
  File "/home/pdunaj/work/ncs/zephyr/scripts/footprint/size_report", line 450, in <module>
    main()
  File "/home/pdunaj/work/ncs/zephyr/scripts/footprint/size_report", line 442, in main
    base + '/', None)
  File "/home/pdunaj/work/ncs/zephyr/scripts/footprint/size_report", line 181, in generate_target_memory_section
    symbols_paths = dict(load_symbols_and_paths(bin_nm, elf_file_abs, source_dir))
  File "/home/pdunaj/work/ncs/zephyr/scripts/footprint/size_report", line 35, in load_symbols_and_paths
    symbol, path = parse_symbol_path_pair(line)
  File "/home/pdunaj/work/ncs/zephyr/scripts/footprint/size_report", line 80, in parse_symbol_path_pair
    assert len(fields) >= 4
AssertionError
ninja: build stopped: subcommand failed.
```

**To Reproduce**
Steps to reproduce the behavior:
1. enable `BT_LL_NRFXLIB`
2. run `ninja rom_report`

**Expected behavior**
Should present a memory report.

**Impact**
annoyance - profiling tool is unusable

**Screenshots or console output**
check above

**Environment (please complete the following information):**
ncs master

**Additional context**
N/A

--- a/scripts/footprint/size_report
+++ b/scripts/footprint/size_report
@@ -54,8 +54,10 @@ def load_symbols_and_paths(bin_nm, elf_file, path_to_strip=""):
 #   (symbol_name, "path/to/file")
 #   or
 #   (symbol_name, "")
+#   or
+#   ("", "")
 #
-#   depending on if the file is found or not
+#   depending on whether the symbol name and the file are found or not
 # }
 def parse_symbol_path_pair(line):
     # Line's output from nm might look like this:
@@ -65,9 +67,10 @@ def parse_symbol_path_pair(line):
     #
     # In general lines look something like this:
     #
-    # 'number number string symbol[\t<absolute_path>:line]
+    # 'number number string[\t<symbol>][\t<absolute_path>:line]
     #
-    # The file is optional, nm might not find out where a symbol came from.
+    # The symbol and file is optional, nm might not find out what a
+    # symbol is named or where it came from.
     #
     # NB: <absolute_path> looks different on Windows and Linux
 
@@ -77,12 +80,20 @@ def parse_symbol_path_pair(line):
 
     fields = line_without_tabs.split()
 
-    assert len(fields) >= 4
+    assert len(fields) >= 3
 
-    symbol = fields[3]
+    # When a symbol has been stripped, it's symbol name does not show
+    # in the 'nm' output, but it is still listed as something that
+    # takes up space. We use the empty string to denote these stripped
+    # symbols.
+    symbol_is_missing = len(fields) < 4
+    if symbol_is_missing:
+        symbol = ""
+    else:
+        symbol = fields[3]
 
-    file_is_missing = len(fields) == 4
 
+    file_is_missing = len(fields) < 5
     if file_is_missing:
         path = ""
     else:

